packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+
+-   We fix an issue regrading native navigation support.
```

Comment:
I see a typo: **regrading** and we should also say: We fix**ed**.

> We fixed an issue regarding native navigation support.

Reply:
@ancajuravle can you check this please?